py/helpers/cache/cache_factory.py

```diff
@@ -0,0 +1,17 @@
+from .cache_gcs import GCSCache
+from .cache_local import LocalCache
+from .cache_redis import RedisCache
+
+
+class CacheFactory:
+
+  @staticmethod
+  def get_cache(cache_type='local', config={}, with_redis=None):
```

Comment:
Config dicts are not currently used in Sight. How do you plan to set this config? Would it make sense to set the local base path, etc. using command line flags or start setting up the full mechanism for configuration objects?

Reply:
as we already have too many flags for sight, adding extras for cache will add more overhead. we can have default config for each case - local, gcs and redis which can be easily overridden by custom params